### PR TITLE
Refactor landing page and add fixes

### PR DESCRIPTION
**Pull request by m0nggh**
This PR is to fix bugs and refactor some code from Stacey's PR!

Question:
Do we want to support querying of live data for the executions and users count?

## Screenshot

https://github.com/opengovsg/plumber/assets/65110268/a61cebcb-ab9a-4efc-8eb7-abe193e05bcb

### CODE REVIEW

File: packages/frontend/src/pages/Landing/Agencies.tsx [hunk 1]
```diff
@@ -22,17 +10,17 @@ import SpfLogo from 'assets/landing/SPF.svg'
 
 export default function Agencies() {
   return (
-    <Box bg="white" py={88}>
+    <Box bg="white" py="88px">
```
Comment (ogp-weeloong) [inline packages/frontend/src/pages/Landing/Agencies.tsx:13]: nit: 

```suggestion
    <Box bg="white" py={22}>
```

File: packages/frontend/src/pages/Landing/components/AppList.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
+export const CUSTOM_APPS = [
```
Comment (ogp-weeloong) [inline packages/frontend/src/pages/Landing/components/AppList.ts]: nit: 

```suggestion
export const CUSTOM_APPS: AppIntegrationProps[] = [
```

File: packages/frontend/src/pages/Landing/components/AppList.ts [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  },
+  {
+    iconName: 'toolbox',
+    name: 'Only continue if',
+    description:
+      'Only allow a Pipe to proceed when a certain condition is met.',
+  },
+]
+
+export const COMMON_APPS = [
```
Comment (ogp-weeloong) [inline packages/frontend/src/pages/Landing/components/AppList.ts]: nit:

```suggestion
export const COMMON_APPS: AppIntegrationProps[] = [
```

File: packages/frontend/src/pages/Landing/components/AppIntegration.tsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    <Flex
+      flexDir="column"
+      borderRadius="md"
+      p={4}
+      bg="secondary.50"
+      gap={4}
+      h="100%"
+    >
+      <Image
+        src={APP_ICON_URL(iconName)}
```
Comment (ogp-weeloong) [inline packages/frontend/src/pages/Landing/components/AppIntegration.tsx:22]: nit: technically not a constant although i know what you're getting at...

```suggestion
        src={appIconUrl(iconName)}
```

File: packages/frontend/src/pages/Landing/Agencies.tsx (hunk 1 above)
Comment (ogp-weeloong) [inline packages/frontend/src/pages/Landing/Agencies.tsx]: nit: 
```suggestion
    <Box bg="white" py="5.5rem">
```

File: packages/frontend/src/pages/Landing/TrainingSession.tsx [hunk 5]
```diff
@@ -1,24 +1,37 @@
...
-      <VStack align="left" maxW="720" gap={5}>
-        <Text textStyle={{ base: 'h3', md: 'heading' }}>
-          Save time for your agency with automations
-        </Text>
-        <Text textStyle={{ base: 'subhead-2', md: 'h6' }}>
-          We are passionate about spreading the benefits of automations
-          throughout WOG. Get started on automations for your agency.
-        </Text>
-        <Button onClick={() => navigate('/go.gov.sg/plumber-ama')} width="50%">
+    <Container py="88px" px={6}>
```
Comment (ogp-weeloong) [inline packages/frontend/src/pages/Landing/TrainingSession.tsx]: nit:

```suggestion
    <Container py="5.5rem" px={6}>
```

File: packages/frontend/src/pages/Landing/ToolsLanding.tsx [hunk 6]
```diff
@@ -1,174 +1,41 @@
...
             </Text>
             <Text textStyle="body-1">
-              Plumber has a variety of in-built tools that can help your Pipes
-              do more.
+              Plumber has a variety of internal tools that can help you do more
+              with your workflow. Set up conditional and branching logic to
+              carry out different workflows.
             </Text>
-          </VStack>
-          <Grid
```
Comment (ogp-weeloong) [inline packages/frontend/src/pages/Landing/ToolsLanding.tsx:66]: nice

**Review by ogp-weeloong — APPROVED** [review]
LGTM but as demonstrated, can we convert the lottie to gif

File: packages/frontend/src/pages/Landing/components/AppIntegration.tsx (hunk 4 above)
Comment (m0nggh) [reply packages/frontend/src/pages/Landing/components/AppIntegration.tsx:22]: discussed to keep it caps based on the file

**Pull request merged**